src/views/ConfigEditor.tsx

```diff
@@ -196,7 +210,9 @@ const ConfigEditor = (props: ConfigEditorProps) => {
           </>
         )}
       </ConfigSection>
-
+      {config.featureToggles['secureSocksDSProxyEnabled' as keyof FeatureToggles] && gte(config.buildInfo.version, '10.0.0') && (
```

Comment:
```suggestion
      {config.secureSocksDSProxyEnabled && (
```

Reply:
Is this the case if you have in your grafana config?
```
[secure_socks_datasource_proxy]
enabled = false
```
This is a separate option, not a feature toggle.

Reply:
got it yeah, I was definitely comparing with the wrong thing.. works now thanks!